Sources/SotoTestUtils/TestUtils.swift

```diff
@@ -113,4 +114,8 @@ public struct TestEnvironment {
         }
         return AWSClient.loggingDisabled
     }()
+
+    public static var context: Context = {
+        DefaultContext.topLevel(logger: logger)
+    }()
```

Comment:
Ah for testing only right? Okey. Would be worried if this was for the real sources, but in testing okey 👍

Reply:
Just for testing